tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "declaration": true,
+    "outDir": "./dist",
+    "strict": true,
```

Comment:
Is it worth adding `eslint` with some recommended strict configs too? My personal pet peeve is allowing `any` type anywhere in the project, so I'd love to see that banned (`strict` TS config takes care of implicit any, but not explicit).